site/content/contribute/plugins/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Plugins"
+date: "2018-03-19T12:01:23-04:00"
+section: "contribute"
+weight: 7
+---
+
+Mattermost plugins are isolated pieces of code, separate from the main repositories, used to extend the functionality of the Mattermost server and webapp. Please see the [Plugin Developer Documentation](/extend/plugins) to learn more.
```

Comment:
I would like the see text about HW ticket getting moved back into this doc to ensure that new community members find it right away.

Reply:
@hanzei The reason I moved it is because the subsections stand out more than the parent category to me. I have a feeling a new contributor will click on the "plus" icon next to "Plugins", as opposed to clicking the "Plugins" tab itself, and not ever see the page that shows explains the HW tickets. If the whole tab was clickable, it would be easier to notice, but you have to click right on the word "Plugins".

Maybe have details in the child tab, and a link to the child tab in the main tab? Here are the docs for example
https://developers.mattermost.com/contribute/getting-started

Also the tab order on the docs is currently
- Getting Started
- Server
- Web App
- Mobile Apps
- Redux
- Desktop App

What do you think about putting the Plugins section above Redux?

Reply:
:+1: for moving `Plugins` before `Redux`.